pkg/hubble/metrics/api/context_test.go

```diff
@@ -189,14 +189,22 @@ func TestParseGetLabelValues(t *testing.T) {
 			"k8s:app=barapp",
 		},
 	}
-	flow := &pb.Flow{Source: sourceEndpoint, Destination: destinationEndpoint, TrafficDirection: pb.TrafficDirection_INGRESS}
+	flow := &pb.Flow{
+		IP: &pb.IP{
+			Source:      "1.2.3.4",
+			Destination: "5.6.7.8",
+		},
+		Source:           sourceEndpoint,
+		Destination:      destinationEndpoint,
+		TrafficDirection: pb.TrafficDirection_INGRESS,
+	}
 	assert.EqualValues(t,
 		mustGetLabelValues(opts, flow),
 		[]string{
-			// source_pod, source_namespace, source_workload, source_app
-			"foo-deploy-pod", "foo-ns", "foo-deploy", "fooapp",
+			// source_ip, source_pod, source_namespace, source_workload, source_app
+			"1.2.3.4", "foo-deploy-pod", "foo-ns", "foo-deploy", "fooapp",
 			// destination_pod, destination_namespace, destination_workload, destination_app
-			"bar-deploy-pod", "bar-ns", "bar-deploy", "barapp",
+			"5.6.7.8", "bar-deploy-pod", "bar-ns", "bar-deploy", "barapp",
```

Comment:
Missing `destination_ip` in the comment above the change?

Reply:
Good catch.